gomint-api/src/main/java/io/gomint/event/world/SignChangeTextEvent.java

```diff
@@ -109,10 +109,10 @@ public String getLine(int line) {
     /**
      * Set the lines the sign should have after the event call
      *
-     * @param lines the lines the sign should have
+     * @param lines the lines the sign should have after the event call
      */
     public void setLines(List<String> lines) {
-        Preconditions.checkArgument(lines.size() != 4, "Sign may only have 4 lines of text");
+        Preconditions.checkArgument(lines.size() != 3, "Sign must have 4 lines of text");
```

Comment:
A list size of 4 indicates elemnts 0 - 3 are present. Checking != 3 is wrong.

Reply:
No it's correct. Count: 0, 1, 2, 3 = 4